released/packages/coq-gappa/coq-gappa.1.4.5/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+maintainer: "guillaume.melquiond@inria.fr"
+homepage: "http://gappa.gforge.inria.fr/"
+dev-repo: "git+https://gitlab.inria.fr/gappa/coq.git"
+bug-reports: "https://gitlab.inria.fr/gappa/coq/issues"
+license: "LGPL-3.0-or-later"
+build: [
+  ["autoconf"] {dev}
+  ["./configure"]
+  ["./remake" "-j%{jobs}%"]
+]
+install: ["./remake" "install"]
+depends: [
+  "ocaml"
+  "coq" {>= "8.8.1" & < "8.14~"}
+  "coq-flocq" {>= "3.0"}
+  "conf-autoconf" {build & dev}
+  ("conf-g++" {build} | "conf-clang" {build})
+]
+tags: [
+  "keyword:floating point arithmetic"
+  "keyword:interval arithmetic"
+  "keyword:decision procedure"
+  "category:Computer Science/Decision Procedures and Certified Algorithms/Decision procedures"
+  "logpath:Gappa"
+  "date:2020-11-26"
+]
+authors: [ "Guillaume Melquiond <guillaume.melquiond@inria.fr>" ]
+synopsis: "A Coq tactic for discharging goals about floating-point arithmetic and round-off errors using the Gappa prover"
+url {
+  src: "https://gforge.inria.fr/frs/download.php/file/38380/gappalib-coq-1.4.5.tar.gz"
```

Comment:
Is this URL really what we want (gforge being shut down)

Reply:
Certainly not in the long run. But some people are (very slowly) coming to the realization that shutting down a service before first proposing a replacement might not have been the wisest move ever (so much for "inventeurs du monde numérique"). So, I am waiting to see what they are coming with before deciding where I host my packages. In case they end up proposing something sensible, I don't want to have to move twice.